go/worker/client/worker.go

```diff
@@ -58,8 +59,13 @@ func (w *Worker) Start() error {
 
 	// Wait for all runtimes to be initialized.
 	go func() {
-		for _, rt := range w.runtimes {
+		for id, rt := range w.runtimes {
 			<-rt.Initialized()
+
+			// After the runtime is finished initializing, enable registration if applicable.
```

Comment:
```suggestion
```

Reply:
Why do you want to remove this comment?

Reply:
The code says it all, the comment is not needed.